README.rst

```diff
@@ -89,6 +79,35 @@ in place:
 
 .. installation-end-marker
 
+Contributing
+------------
+
+Ocean's `contributing guide <https://docs.ocean.dwavesys.com/en/stable/contributing.html>`_
+has guidelines for contributing to Ocean packages.
+
+dimod includes some formatting customization in the
+`.clang-format <.clang-format>`_ and `setup.cfg <setup.cfg>`_ files.
+
+Reno
```

Comment:
Can we have the header "Release Notes" instead? That will be self-explanatory also to users who are not familiar with this particular package